config/settings.yml

```diff
@@ -0,0 +1,8 @@
+plugins:
+  debtcollective_theme_enabled:
+    default: true
+    client: true
+  debtcollective_header_links:
+    default: "Dispute Your Debt, Dispute Your Debt, https://tools.debtcollective.org, vdo, blank, remove|Campaigns, Campaigns, /latest, vdo, self, remove|Events, Events, /latest, vdo, self, remove"
```

Comment:
Can we not use an array of yaml objects to do this semantically instead of with these comma separated lists?

Also Campaigns and Events are both just pointing to `/latest`. Shouldn't they point somewhere more specific?

Reply:
This is the default configuration, will be changed when we deploy with terraform. I can remove the other items and just keep the Dispute Your Debt one, the idea here was showing the format instead of being a safe default.

About the list, I would prefer to use an array too, but the Discourse API saves lists like this (a string where values are separated by `|`), so we are limited by that. ex: https://github.com/discourse/discourse_api_docs/blob/master/responses/admin/site_settings.json#L436-L455

I added a new screenshot to show how the settings looks in the admin section